packages/features/eventtypes/components/event-type/tabs/team/EventTeamTab.tsx

```diff
@@ -12,7 +11,12 @@ import AddMembersWithSwitch, {
 import AssignAllTeamMembers from "@calcom/features/eventtypes/components/AssignAllTeamMembers";
 import ChildrenEventTypeSelect from "@calcom/features/eventtypes/components/ChildrenEventTypeSelect";
 import { sortHosts, weightDescription } from "@calcom/features/eventtypes/components/HostEditDialogs";
-import type { FormValues, TeamMember } from "@calcom/features/eventtypes/lib/types";
+import type {
```

Comment:
In the UI we call this tab "assignment" and in query params "tabName=team" - i am wondering if its better to call this component EventTeamAssignment and the tab from team to "assignment" because that is what it does and how it appears in UI.